satellizer.js

```diff
@@ -850,20 +853,21 @@ if (typeof module !== 'undefined' && typeof exports !== 'undefined' && module.ex
       })();
 
       if (!isStorageAvailable) {
-        $log.warn('Satellizer Warning: ' + config.storageType + ' is not available.');
+        console.warn('Satellizer Warning: ' + config.storageType + ' is not available.');
```

Comment:
When you pulled the latest PR you should have had a merge conflict. Should be $log.warn here.